cmd/convox/install.go

```diff
@@ -87,7 +87,7 @@ func init() {
 			},
 			cli.StringFlag{
 				Name:   "build-instance",
-				Value:  "",
+				Value:  "t2.small",
```

Comment:
This will add a default value of `t2.small` which will each time set the BuildInstance parameter in Cloud formation. I would remove that and leave only the rack.json option.

Reply:
Yes, by default this will cause `convox install` to set the `BuildInstance` parameter to `t2.small`, unless it is overridden with `convox install --build-instance ""`. 

The current behavior is to set the default value of `BuildInstance` in `rack.json`, but the default behavior cannot be overridden during the `convox install` stage.

Reply:
I would also be in favor of removing the default build instance, if that's an option. (Setting both values to `""`.) I think the majority of users probably don't want to run a dedicated instance for builds (but I might be wrong.)

Reply:
This relates to following commit: https://github.com/convox/rack/commit/e39400a314e8938dd1768518c4a944228cc90553

Reply:
The default is now for a BuildInstance as the experience is significantly improved with one. I think it's appropriate to keep that default in the CloudFormation stack.

I would be amenable to a `--no-build-instance` flag to `convox install`

Reply:
`--no-build-instance` works for me!